airbyte-webapp/src/components/ui/ListBox/ListBox.tsx

```diff
@@ -0,0 +1,73 @@
+import { Listbox } from "@headlessui/react";
+import classNames from "classnames";
+import React from "react";
+
+import styles from "./ListBox.module.scss";
+
+export interface ListBoxControlButtonProps<T> {
+  selectedOption: Option<T>;
+}
+
+// Not using arrow function due to generic type
+// eslint-disable-next-line react/function-component-definition
+function DefaultControlButton<T>({ selectedOption }: ListBoxControlButtonProps<T>) {
```

Comment:
```suggestion
```suggestion
const DefaultControlButton = <T,>({ selectedOption }: ListBoxControlButtonProps<T>) => {
```

Note the trailing comma behind the `T` is needed since this is a `tsx` file and otherwise the TS parser will think you're trying to open a JSX element in this case.
```